html/js/product-multilingual.js

```diff
@@ -41,1076 +41,1077 @@ var imagefield_url = {};
+            $th.attr('lang', newID);
+        });
 
-    $clone.find('.tab_language').each(function() {
+        $clone.find('.tab_language').each(function() {
 
-      $(this).html(language);
-    });
+            $(this).html(language);
+        });
 
-    $clone.insertBefore($(this));
+        $clone.insertBefore($(this));
 
-    $clone.addClass('active').removeClass('new_lc').removeClass('hide');
+        $clone.addClass('active').removeClass('new_lc').removeClass('hide');
 
-    $(".select_crop").filter(":visible").selectcrop('init');
-    $(".select_crop").filter(":visible").selectcrop('show');
+        $(".select_crop").filter(":visible").selectcrop('init');
+        $(".select_crop").filter(":visible").selectcrop('show');
 
-  });
+    });
 
-  update_move_data_and_images_to_main_language_message();
+    update_move_data_and_images_to_main_language_message();
 
-  $(document).foundation('tab', 'reflow');
+    $(document).foundation('tab', 'reflow');
 }
 
-function select_nutriment(event,ui) {
-
-
-	//alert(ui.item.id + ' = ' + ui.item.value);
-	//alert($("#add_nutriment").val());
-	var id = $(this).attr('id');
-	id = id.replace("_label", "");
-	$('#' + id).focus();
-	$('#' + id + '_unit').val(ui.item.unit);
-	var unit = (ui.item.unit == '%' ? '%' : ui.item.unit).toLowerCase();
-	var unitElement = $('#' + id + '_unit');
-	var percentElement = $('#' + id + '_unit_percent');
-	if (unit === '') {
-		unitElement.hide();
-		percentElement.hide();
-	}
-	else if (unit == '%') {
-		unitElement.hide();
-		percentElement.show();
-	}
-	else {
-		unitElement.show();
-		percentElement.hide();
-
-		for (var entryIndex = 0; entryIndex < units.length; ++entryIndex) {
-			var entry = units[entryIndex];
-			for (var unitIndex = 0; unitIndex < entry.length; ++unitIndex) {
-				var unitEntry = entry[unitIndex].toLowerCase();
-				if (unitEntry == unit) {
-					var domElement = unitElement[0];
-					domElement.options.length = 0; // Remove current entries.
-					for (var itemIndex = 0; itemIndex < entry.length; ++itemIndex) {
-						var unitValue = entry[itemIndex];
-						domElement.options[domElement.options.length] = new Option(unitValue, unitValue, false, unitValue.toLowerCase() == unit);
-					}
-
-					if (ui.item.iu) {
-						domElement.options[domElement.options.length] = new Option('IU', 'IU', false, 'iu' == unit);
-					}
-
-					return;
-				}
-			}
-		}
-	}
+function select_nutriment(event, ui) {
+
+
+    //alert(ui.item.id + ' = ' + ui.item.value);
+    //alert($("#add_nutriment").val());
+    var id = $(this).attr('id');
+    id = id.replace("_label", "");
+    $('#' + id).focus();
+    $('#' + id + '_unit').val(ui.item.unit);
+    var unit = (ui.item.unit == '%' ? '%' : ui.item.unit).toLowerCase();
+    var unitElement = $('#' + id + '_unit');
+    var percentElement = $('#' + id + '_unit_percent');
+    if (unit === '') {
+        unitElement.hide();
+        percentElement.hide();
+    } else if (unit == '%') {
```

Comment:
It seems to me you don't use this file any more any where. Should we keep it ?

Reply:
We still use it, it's loaded by /cgi/product.pl